includes/server/class-llms-rest-memberships-controller.php

```diff
@@ -0,0 +1,590 @@
+				),
+				'rendered' => array(
+					'description' => __( 'Rendered message content.', 'lifterlms' ),
+					'default'     => __(
+						'You must belong to the <a href="https://example.com/membership/gold">Gold</a> membership ' .
+						'to access this content.',
+						'lifterlms' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+			),
+		);
+		$schema['properties']['restriction_page_id'] = array(
+			'description' => __(
+				'WordPress page ID used for redirecting non-members when `restriction_action` is `page`.',
+				'lifterlms' ),
+			'type'        => 'integer',
+			'context'     => array( 'view', 'edit' ),
+			'arg_options' => array(
+				'sanitize_callback' => 'absint',
+			),
+		);
+		$schema['properties']['restriction_url']     = array(
+			'description' => __(
+				'URL used for redirecting non-members when `restriction_action` is `custom`.',
+				'lifterlms' ),
+			'type'        => 'string',
+			'context'     => array( 'view', 'edit' ),
+			'format'      => 'uri',
+			'arg_options' => array(
+				'sanitize_callback' => 'esc_url_raw',
+			),
+		);
+		$schema['properties']['sales_page_page_id']  = array(
+			'description' => __(
+				'The WordPress page ID of the sales page. Required when `sales_page_type` equals `page`. ' .
+				'Only returned when the `sales_page_type` equals `page`.',
+				'lifterlms' ),
+			'type'        => 'integer',
+			'context'     => array( 'view', 'edit' ),
+			'arg_options' => array(
+				'sanitize_callback' => 'absint',
+			),
+		);
+		$schema['properties']['sales_page_type']     = array(
+			'description' => __(
+				'Defines alternate content displayed to visitors and non-enrolled students when accessing the post. ' .
+				'- `none` displays the post content. ' .
+				'- `content` displays alternate content from the `excerpt` property.' .
+				'- `page` redirects to the WordPress page defined in `content_page_id`.' .
+				'- `url` redirects to the URL defined in `content_page_url`.',
+				'lifterlms' ),
+			'type'        => 'string',
+			'default'     => 'none',
+			'enum'        => array_keys( llms_get_sales_page_types() ),
+			'context'     => array( 'view', 'edit' ),
+		);
+		$schema['properties']['sales_page_url']      = array(
+			'description' => __(
+				'The URL of the sales page content. Required when `sales_page_type` equals `url`. ' .
+				'Only returned when the `sales_page_type` equals `url`.',
+				'lifterlms' ),
+			'type'        => 'string',
+			'context'     => array( 'view', 'edit' ),
+			'format'      => 'uri',
+			'arg_options' => array(
+				'sanitize_callback' => 'esc_url_raw',
+			),
+		);
+		$schema['properties']['tags']                = array(
+			'description' => __( 'List of membership tags.', 'lifterlms' ),
+			'type'        => 'array',
+			'items'       => array(
+				'type' => 'integer',
+			),
+			'context'     => array( 'view', 'edit' ),
+		);
+
+		return $schema;
```

Comment:
Could you please add a filter here?
https://github.com/eri-trabiccolo/lifterlms-rest/blob/memberships/includes/server/class-llms-rest-memberships-controller.php#L272

See https://github.com/gocodebox/lifterlms-rest/issues/85#issuecomment-528481638
Basically since then I've planned to add filters/action hooks to the other controllers too.